charts/rancher-vsphere-csi/templates/node/windows-daemonset.yaml

```diff
@@ -57,7 +57,7 @@ spec:
             - name: ADDRESS
               value: 'unix://C:\\csi\\csi.sock'
             - name: DRIVER_REG_SOCK_PATH
-              value: {{ .Values.csiNode.prefixPath }}'\\var\\lib\\kubelet\\plugins\\csi.vsphere.vmware.com\\csi.sock'
+              value: '{{ .Values.csiNode.prefixPathWindows | default "C:" }}\\var\\lib\\kubelet\\plugins\\csi.vsphere.vmware.com\\csi.sock'
```

Comment:
If a default value is required, why not put it in the value.yaml file?

Reply:
We can hard code the path directly in the windows-daemonset.yaml file, but this would account for anyone who may have a different path defined. Likely a very specific edge case, but not harmful to give the option to modify if needed. 

I am fine with either direction personally.

Reply:
I mean the `value.yaml` file not the `windows-daemonset.yaml` file, but I think it is also ok to keep it the current way.

Reply:
I like the current approach - the common pattern for helm charts is having an empty or no value in the values.yaml, and handling defaulting in the template. which is what we're doing here.